sql/_13_issues/_12_2h/cases/bug_bts_10105.sql

```diff
@@ -10,7 +10,7 @@ select * from t1;
 
 drop table if exists t1,t2;
 
-create table t2 as SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A  SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A  ;;
+create table t2 as SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A  SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A ;
```

Comment:
It should be divided into two sentences as follows.

create table t2 as SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A;
SELECT '1' A UNION SELECT '2 ' A UNION SELECT '3 ' A UNION SELECT '4 ' A ;

Reply:
The review has been reflected.